.github/workflows/no-response.yml

```
@@ -0,0 +1,23 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```

Wouldn't it be sufficient to run once a day?

```suggestion
    - cron: '5 0 * * *'
```